src/EdFi.AnalyticsMiddleTier.DataStandard32/Asmt/PostgreSQL/0024-View-StudentAssessmentFact-Alter.sql

```diff
@@ -0,0 +1,170 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+DROP VIEW IF EXISTS analytics.asmt_StudentAssessmentFact;
+
+CREATE OR REPLACE VIEW analytics.asmt_StudentAssessmentFact 
+AS
+SELECT
+	CONCAT(
+		StudentAssessment.AssessmentIdentifier,
+		'-', StudentAssessment.Namespace, 
+		'-', StudentAssessment.StudentAssessmentIdentifier, 
+		'-', StudentAssessmentScoreResult.AssessmentReportingMethodDescriptorId,
+		'-', AssessmentPerformanceLevel.PerformanceLevelDescriptorId,
+		'-', StudentAssessmentStudentObjectiveAssessment.IdentificationCode,
+		'-', StudentAssessmentStudentObjectiveAssessmentScoreResult.AssessmentReportingMethodDescriptorId,
+		'-', StudentAssessmentStudentObjectiveAssessmentPerformanceLevel.PerformanceLevelDescriptorId,
+		'-', Student.StudentUniqueId,
+		'-', StudentSchoolAssociation.SchoolId,
+		'-', TO_CHAR(StudentSchoolAssociation.EntryDate, 'yyyymmdd')
+	) AS StudentAssessmentFactKey,
+	CONCAT(
+		StudentAssessment.AssessmentIdentifier, 
+		'-', StudentAssessment.Namespace, 
+		'-', StudentAssessment.StudentAssessmentIdentifier, 
+		'-', Student.StudentUniqueId,
```

Comment:
There is a syntax error here. You have to remove the comma.

Reply:
Good catch, Done!